test/transparentproxy/testdata/alpine-3-default.ip6tables-legacy.golden

```diff
@@ -0,0 +1 @@
+executable not found
```

Comment:
🤔 - so should we remove the test altogether or install the binary? or is the binary not available at all?